serverless-functions/profile_retrieval/package.json

```diff
@@ -11,6 +11,7 @@
   "license": "ISC",
```

Comment:
i wonder if we should have package.json just recreated by sls every time to avoid pinning deps

Reply:
The only reason there's package.json in there at all is for serverless-framework plugins.